src/Gaffer/PlugAlgo.cpp

```diff
@@ -774,6 +774,16 @@ bool canSetValueFromData( const ValuePlug *plug, const IECore::Data *value )
 		case Gaffer::V2fPlugTypeId:
 		case Gaffer::V2iPlugTypeId:
 			return canSetCompoundNumericPlugValue( value );
+		case Gaffer::Color3fVectorDataPlugTypeId:
```

Comment:
We should mention this in the API section of Changes.md

Reply:
done